database/entity/0044-adapt_users_table_for_gradidoid/User.ts

```diff
@@ -0,0 +1,109 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, DeleteDateColumn } from 'typeorm'
+
+@Entity('users', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class User extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({
+    name: 'gradido_id',
+    length: 36,
+    nullable: true,
```

Comment:
```suggestion
    nullable: false,
```

Reply:
yes if the solution with default UUID() will work, it can be set to not nullable